src/core/p5.Renderer.js

```diff
@@ -159,6 +159,7 @@ class Renderer extends p5.Element {
     }
 
     const region = new p5.Image(w, h);
```

Comment:
Do we need to make it it a `new p5.Image(w*pd, h*pd)`? I think this currently still reduces the number of pixels if the graphic is higher density. I think we'd also need up update line 165 to be something like:
```js
drawImage(canvas, x, y, w * pd, h * pd, 0, 0, w * pd, h * pd)
```

Reply:
Awesome! I get what you're saying. I've made the changes in the commit. Take a look now—do you think it looks good? Super open to any further tweaks or suggestions!